public/docs/ts/latest/tutorial/toh-pt3.jade

```diff
@@ -126,7 +126,7 @@ code-example(format="." language="bash").
 code-example(format=".").
   &lt;my-hero-detail [hero]="selectedHero">&lt;/my-hero-detail>
 :marked
-  Notice that the `hero` property is the ***target*** of a property binding &mdash; it's in square brackets to the left of the (=).
+  Notice that the `hero` property is the ***target*** of a property binding &mdash; it's in square brackets to the left of the `=`.
```

Comment:
The `=` is harder to read than (=).  I've gone back and forth on this myself and come to that conclusion. Please retract